optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/PartitionSolver.java

```diff
@@ -72,6 +73,16 @@ public boolean addProblemFactChanges(List<ProblemFactChange<Solution_>> problemF
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public boolean addProblemChange(ProblemChange<Solution_> problemChange) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public boolean addProblemChanges(List<ProblemChange<Solution_>> problemChangeList) {
+        throw new UnsupportedOperationException();
```

Comment:
Sounds like a JIRA for future improvements. Or is there an actual reason why we can not support this here?

Reply:
Good question; I am just mimicking what we already have for the existing PFCs. @ge0ffrey do you happen to know why we don't support PFC with partitioned solving?